packages/driver/src/adapter.ts

```diff
@@ -1,20 +1,23 @@
-import * as crypto from "crypto";
-import { promises as fs } from "fs";
-import * as net from "net";
-import * as os from "os";
-import * as path from "path";
-import * as tls from "tls";
+import * as crypto from "node:crypto";
+import { promises as fs } from "node:fs";
+import net from "node:net";
+import os from "node:os";
+import path from "node:path";
+import * as tls from "node:tls";
 
-import process from "process";
-import * as readline from "readline";
-import { Writable } from "stream";
+import process from "node:process";
+import * as readline from "node:readline";
+import { Writable } from "node:stream";
 
 export { path, net, fs, tls, process };
 
 export async function readFileUtf8(...pathParts: string[]): Promise<string> {
   return await fs.readFile(path.join(...pathParts), { encoding: "utf8" });
 }
 
+// export function hasFSReadPermission(): boolean {
+//   return Deno.permissions.querySync({ name: "read" }).state === "granted";
+// }
```

Comment:
This should be removed

Reply:
I left this comment to ask do we need this, and I believe we need it for Deno users, so I added it in the hasFSReadPermission func